<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StmWindow</class>
 <widget class="QDialog" name="StmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STM</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background: rgba(214, 219, 233,255);
}

QComboBox{
background: rgba(214, 219, 233,255);
border: none;
}

QComboBox:hover{
background: rgba(255, 255, 191,255);
border:  2px solid rgba(255, 170, 0,255);
}

QPushButton:hover{
background: rgba(255, 255, 191,255);
border:  2px solid rgba(255, 170, 0,255);
}

QPushButton{
border: none;
background: rgba(214, 219, 233,255);
}

QTableWidget {
    background: rgba(214, 219, 233,255);
border: none;
}

QHeaderView::section{
background-color: rgba(235, 241, 255, 255);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,3">
   <item>
    <widget class="QLabel" name="stmOutput">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>15</horstretch>
       <verstretch>15</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1,1,2,1,2,2,2,2,1,1">

    <item>
      <widget class="QLabel" name="ColorMap">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>colormap</string>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>15</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxCMaps">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>

    <item>
    <layout class="QHBoxLayout" name="horizontalLayout1" stretch="2,2">
     <item>
      <widget class="QPushButton" name="pushButtonCMin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>cMin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCMax">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>cMax</string>
       </property>
      </widget>
     </item>
    </layout>
    </item>


   <item>
    <widget class="QLabel" name="labelCutoffs">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>colormap cutoff</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CustomTableWidget" name="tableWidgetCutoff">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>

     <item>
      <widget class="QLabel" name="STMParams">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>STM-Sim parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetstmParams">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLabel" name="MTip">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>tip magnetization</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetMTip">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLabel" name="Contrast">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Contrast scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomTableWidget" name="tableWidgetContrast">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>

     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomTableWidget</class>
   <extends>QTableWidget</extends>
   <header>CustomTableWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>readparams()</slot>
 </slots>
</ui>
